task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,7 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return !(num % 3) ? !(num % 5) ? 'FizzBuzz' : 'Fizz' : !(num % 5) ? 'Buzz' : num;
```

Comment:
Multiple ternary operators are difficult to read. It's better to use if-statements or at least put each ternary operator in parenthesis